recipes/nipype/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "0.12.0" %}
+
+package:
+  name: nipype
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/nipy/nipype/archive/{{ version }}.tar.gz
+  md5: 4d010bae6028cf4622bc1946c679fabc
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I guess it's not my day. Sorry about this. Let's drop the arguments here and drop `setuptools`.